graylog2-server/src/main/java/org/graylog2/plugin/Version.java

```diff
@@ -217,6 +243,12 @@ public static Version fromClasspathProperties(String path, String propertyName,
         return defaultVersion;
     }
 
+    private static URL getResource(Class<?> clazz, String path) {
+        final URL url = requireNonNull(clazz, "Class argument is null!").getClassLoader().getResource(path);
```

Comment:
> There's also a variant of Resources#getResource() for this.

I tried the `Resources#getResource()` variant but it didn't work... No idea why.

I will try again.